src/parser/block-additions/core-image.php

```diff
@@ -1,21 +1,38 @@
 <?php
+/**
+ * Enhancements for the core/image block
+ * 
+ * @package vip-block-data-api
+ */
 
 namespace WPCOMVIP\BlockDataApi\ContentParser\BlockAdditions;
 
 defined( 'ABSPATH' ) || die();
 
+/**
+ * Enhance the core/image block with attributes related to its size.
+ */
 class CoreImage {
+	/**
+	 * Initialize the CoreImage class.
+	 * 
+	 * @access private
+	 */
 	public static function init() {
 		add_filter( 'vip_block_data_api__sourced_block_result', [ __CLASS__, 'add_image_metadata' ], 5, 4 );
 	}
 
 	/**
-	 * @param array[string]array $sourced_block
-	 * @param string $block_name
-	 * @param int $post_id
-	 * @param array[string]array $block
+	 * Add size metadata to core/image blocks
+	 * 
+	 * @param array    $sourced_block the sourced block result.
+	 * @param string   $block_name the name of the block.
+	 * @param int|null $post_id the id of the post.
+	 * @param array    $block the block itself.
+	 * 
+	 * @access private
 	 *
-	 * @return array[string]array
+	 * @return array the updated sourced block, with the new metadata information
```

Comment:
Nit - I think these look better as individual capitalized "sentences", dropping the "the". For example:

```suggestion
	 * @return array Updated sourced block, with the new metadata information
```

Same goes for parameters, for example:

```
@param array $sourced_block Sourced block result.
```

Reply:
Yip, I get what you mean. Making that change rn